exercises/practice/proverb/.meta/Proverb.example.ps1

```diff
@@ -0,0 +1,38 @@
+Function Get-Proverb() {
+    <#
+    .SYNOPSIS
+    Given a list of inputs, generate the relevant proverb.
+
+    .DESCRIPTION
+    Take a list of inputs and output the full text of a proverbial rhyme base on those inputs.
+
+    .PARAMETER Data
+    The list of inputs to generate the proverb.
+
+    .EXAMPLE
+    Get-Proverb -Input @("nail", "shoe", "horse", "rider", "message", "battle", "kingdom")
```

Comment:
Should say `Get-Proverb -Data @("nail", "shoe", "horse", "rider", "message", "battle", "kingdom")`